.github/workflows/build.yml

```diff
@@ -1,5 +1,6 @@
 name: Build dlc.dat
 on:
+  workflow_dispatch:
```

Comment:
Do one thing in a single PR.

Reply:
Why did you want to add this setting? Which repo did you want to trigger by this workflow?